warp/aggregator/aggregator_test.go

```diff
@@ -23,7 +23,7 @@ func newValidator(t testing.TB, weight uint64) (*bls.SecretKey, *avalancheWarp.V
 	pk := bls.PublicFromSecretKey(sk)
 	return sk, &avalancheWarp.Validator{
 		PublicKey:      pk,
-		PublicKeyBytes: bls.PublicKeyToBytes(pk),
+		PublicKeyBytes: bls.PublicKeyToCompressedBytes(pk),
```

Comment:
Why is this changed?

Reply:
`PublicKeyToBytes` doesn't exist anymore. It was renamed to `PublicKeyToCompressedBytes` to avoid confusion.

see: https://github.com/ava-labs/avalanchego/commit/ff4bf3fbec8a1f4696f8af2047c8fe1cd0a24589#diff-fc441238ff8082ab78adf1dc73b931d565c0ed7b03714bc906cd8b0b5a3d3304R28